effectiveOfficeBackend/Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM gradle:7-jdk11 AS build
+FROM ghcr.io/graalvm/jdk-community:22.0.1 AS build
 COPY . /home/gradle/src
 WORKDIR /home/gradle/src
 RUN ./gradlew buildFatJar --no-daemon
 
-FROM openjdk:11
+FROM build
```

Comment:
For what reason you using the build stage, with the source code and all the garbage that appeared after the jar file build?

It was also necessary to build runtime image from a clean graalvm image